components/navigation.css

```diff
@@ -0,0 +1,16 @@
+.navbar-container{
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    padding: 1.5rem 2.5rem;
+    max-height: 3.75rem;
+    width: 100%;
+    align-items: center;
```

Comment:
one suggestion here, have all the flex related properties grouped together, helps to understand what all things flex is doing to a element at a glance

Reply:
Will push the changes at the end else there will be so many merge conflicts